2_segmenting_data/Segmented_Images/D6_1_cyto-segmented.tsv

```diff
@@ -0,0 +1 @@
+
```

Comment:
hmm, some of these look empty. This could be a red flag. Does this mean there are no cells in the image?

Reply:
It would mean either all cytoplasm are touching the edges or no nuclei could be found. After checking that image there should only be one cytoplasm. After rerunning the segmentation with the new parameters, I hope to see one cytoplasm in this table.

Reply:
Just reran it, this issue is now fixed and there is one cytoplasm and one nucleus!